### DISCUSSION TITLE
Is possibile to enable hardware acceleration on ffmpeg during download?

### DISCUSSION THREAD

**Discussion by tmm360** [Tyrrrz/YoutubeExplode#694] [Q&A]
I don't see any way to enable it or to pass custom arguments. Thanks.

**Comment by Tyrrrz** [comment]
Unless it's on by default, probably no. Is there some specific switch/option you're interested in?

**Reply by tmm360** [reply]
Yes, I've found this article from nvidia: https://docs.nvidia.com/video-technologies/video-codec-sdk/ffmpeg-with-nvidia-gpu/index.html

Here it says we have to pass `-hwaccel cuda` (if cuda is the used hw accelerator drivers), `-hwaccel_output_format cuda` and use `-c:v h264_nvenc` as video encoder. Obviously the documentation have to be studied more in deep.

Eventually I could help providing a PR if it could be accepted.
